situp-api/src/main/java/com/amazon/situp/model/buffer/AbstractBuffer.java

```diff
@@ -0,0 +1,87 @@
+package com.amazon.situp.model.buffer;
```

Comment:
Everything currently is being renamed as per new legal name - This would have lot of conflicts